index.js

```diff
@@ -1,20 +1,130 @@
 function ReadElement(selector) {
   return document.querySelector(selector);
 }
-
 const menu = ReadElement('#menu-icon');
 const nav = ReadElement('.mobile-nav');
 const closeNav = ReadElement('.close-button');
 const choose = ReadElement('.mobile-nav');
-
+const viewModal1 = ReadElement('.view-card-1');
+const viewModal2 = ReadElement('.view-card-2');
+const viewModal3 = ReadElement('.view-card-3');
+const viewModal4 = ReadElement('.view-card-4');
+const CardModal = ReadElement('.modal-section');
```

Comment:
- [ ] Please let's be consistent according to JS best practises, this constant variable name suppose to be in camelCase I think. 👍

Reply:
@OyePriscilla Please are you referring to the CardModa? Because all the other variable names are in camelCase